src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -160,8 +160,14 @@ def _register_run_job_task(self, graph: DependencyGraph):  # pylint: disable=unu
     def _register_pipeline_task(self, graph: DependencyGraph):  # pylint: disable=unused-argument
         if not self._task.pipeline_task:
             return
-        # TODO: https://github.com/databrickslabs/ucx/issues/1638
-        yield DependencyProblem('not-yet-implemented', 'Pipeline task is not yet implemented')
+
+        pipeline = self._ws.pipelines.get(self._task.pipeline_task.pipeline_id)
+        pipeline_libraries = pipeline.spec.libraries
+        for library in pipeline_libraries:
+            if library.notebook:
```

Comment:
What about the wheel library? Also log the cases you've skipped

Reply:
Pipeline library has only these 4 options:
https://github.com/databricks/databricks-sdk-py/blob/main/databricks/sdk/service/pipelines.py#L1234